Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about scuba diving?"
+    m 3eub "after immersion in water, you can see a wonderful underwater world full of diverse marine life."
```

Comment:
```suggestion
    m 3eub "After immersion in water, you can see a wonderful underwater world full of diverse marine life."
```
Before this line, I think there should be a brief explanation on what scuba diving is. for example "When you go diving underwater with scuba gear". 

It doesn't sound right to me when you say "Have you heard about X?" and then jump straight into talking about it as if the player already knows.